ArcGISRuntimeSDKQt_QMLSamples/Features/FeatureLayer_DefinitionExpression/FeatureLayer_DefinitionExpression.qml

```diff
@@ -77,22 +77,62 @@ Rectangle {
         }
         spacing: 5
 
+        Label {
+
+            text: "Current feature count: " + updateFeatureCount();
+        }
         // button to apply a definition expression
         Button {
             text: "Apply Expression"
+            width: 200
             enabled: featureTable.loadStatus === Enums.LoadStatusLoaded
             onClicked: {
                 featureLayer.definitionExpression = "req_Type = \'Tree Maintenance or Damage\'"
             }
         }
+        // button to apply display filter
+        Button {
+            text: "Apply Display Filter"
+            width: 200
+            enabled: featureTable.loadStatus === Enums.LoadStatusLoaded
+            onClicked: {
+                if (featureLayer.loadStatus === Enums.LoadStatusLoaded)
+                {
+                    var displayFilter = ArcGISRuntimeEnvironment.createObject("DisplayFilter", {name: "Damaged Trees", filterId: "Damaged Trees", whereClause: "req_Type = \'Tree Maintenance or Damage\'"});
+                    var displayFilterDefintionVar = ArcGISRuntimeEnvironment.createObject("ManualDisplayFilterDefinition");
+                    displayFilterDefintionVar.availableFilters.append(displayFilter);
+                    displayFilterDefintionVar.activeFilter = displayFilter;
+
+                    featureLayer.displayFilterDefinition = displayFilterDefintionVar;
+                }
```

Comment:
If `enabled` is controlled by `featureTable.loadStatus === Enums.LoadStatusLoaded` then there shouldn't be a case where `onClicked` fires when the feature table isn't loaded, right? Therefore we should be able to just omit it.
```suggestion
            enabled: featureTable.loadStatus === Enums.LoadStatusLoaded
            onClicked: {
                var displayFilter = ArcGISRuntimeEnvironment.createObject("DisplayFilter", {name: "Damaged Trees", filterId: "Damaged Trees", whereClause: "req_Type = \'Tree Maintenance or Damage\'"});
                var displayFilterDefintionVar = ArcGISRuntimeEnvironment.createObject("ManualDisplayFilterDefinition");
                displayFilterDefintionVar.availableFilters.append(displayFilter);
                displayFilterDefintionVar.activeFilter = displayFilter;

                featureLayer.displayFilterDefinition = displayFilterDefintionVar;
```